src/entities/euring-codes/place-code-entity.ts

```diff
@@ -2,10 +2,16 @@ import { Entity, PrimaryColumn, Column, OneToMany } from 'typeorm';
 import { Length, IsAlphanumeric, IsOptional, IsString } from 'class-validator';
 import { equalLength } from '../../validation/validation-messages';
 import { Ring } from '../ring-entity';
+import { EntityDto } from '../common-interfaces';
+
+export interface PlaceCodeDto extends EntityDto {
+  country?: string | null;
```

Comment:
The EURING's DTO definitions use an inconsistent approach.  Sometimes they contain an `id`, sometimes they don' t. While all the codes have an `id` and it has the widest use.

Also, some EURING entities export DTO, some do not.

I would like to know the reasons why this is done.

Reply:
Seems that I got you. Try to not extend interfaces `EntityDto` plus `WithDescription` or do it in other way.

Reply:
Resolving